### PR TITLE
Example of how to use Number in Typescript

### PR DESCRIPTION
**Pull request by Coldplayer1995**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Review by amandeepmittal — APPROVED** [review]

File: guide/english/typescript/number-type/index.md [hunk 1]
```diff
@@ -12,4 +12,21 @@ let decimal: number = 7;
 let hex: number = 0xf00d;
 let binary: number = 0b0110;
 let octal: number = 0o531;
-```
\ No newline at end of file
+```
+
+Example of usage Number type in typescript:
```
Comment (itaisteinherz) [inline guide/english/typescript/number-type/index.md]:
```suggestion
Example usage of the type `Number` in TypeScript:
```

**Review by itaisteinherz — CHANGES_REQUESTED** [review]
This looks good. My only suggestion other than my comment is to properly format and indent the code, in order for it to be more readable.

File: guide/english/typescript/number-type/index.md [hunk 2]
```diff
@@ -12,4 +12,22 @@ let decimal: number = 7;
...
 let binary: number = 0b0110;
 let octal: number = 0o531;
-```
\ No newline at end of file
+```
+
+Example of Number type usage in TypeScript:
+
+```typescript
+class Employee extends Person{
```
Comment (itaisteinherz) [inline guide/english/typescript/number-type/index.md]:
```suggestion
class Employee extends Person {
```

File: guide/english/typescript/number-type/index.md [hunk 3]
```diff
@@ -12,4 +12,22 @@ let decimal: number = 7;
...
+  age:Number;
```
Comment (itaisteinherz) [inline guide/english/typescript/number-type/index.md]:
```suggestion
  age: Number;
```

File: guide/english/typescript/number-type/index.md [hunk 4]
```diff
@@ -12,4 +12,22 @@ let decimal: number = 7;
...
+  constructor(age : Number){
```
Comment (itaisteinherz) [inline guide/english/typescript/number-type/index.md]:
```suggestion
  constructor(age: Number) {
```

File: guide/english/typescript/number-type/index.md [hunk 5]
```diff
@@ -12,4 +12,22 @@ let decimal: number = 7;
...
+    //calling constructor of Person
```
Comment (itaisteinherz) [inline guide/english/typescript/number-type/index.md]:
```suggestion
    // Run when an object of type `Person` is created
```

File: guide/english/typescript/number-type/index.md [hunk 6]
```diff
@@ -12,4 +12,22 @@ let decimal: number = 7;
...
+    super();
+    this.age = age;
+  }
+ 
+  getAge(): Number{
```
Comment (itaisteinherz) [inline guide/english/typescript/number-type/index.md]:
```suggestion
  getAge(): Number {
```

**Review by itaisteinherz — CHANGES_REQUESTED** [review]
@cmccormack I've added a few more suggestions to further improve the code quality.

**Comment by cmccormack** [comment]
This looks good to me.  Thank you for your contribution!

**Pull request merged**